src/runtime/executor.coffee

```diff
@@ -23,17 +23,17 @@ module.exports.Executor = class Executor
       patientSource.nextPatient()
     r
 
-  exec: (patientSource) ->
-    Results r = @exec_patient_context(patientSource)
+  exec: (patientSource, executionDateTime) ->
```

Comment:
There are a couple of examples in the source code that invoke this function.  Can you update them to pass in the `executionDateTime`?  I think it would be good to show this:

https://github.com/cqframework/cql-execution/blob/ea6806eccf059722ded252d1a66a4ef535280f3d/src/example/exec-age.coffee#L46

https://github.com/cqframework/cql-execution/blob/ea6806eccf059722ded252d1a66a4ef535280f3d/src/example/exec-cms146v2_CQM.coffee#L102